pkg/provision/providers/qemu/arch.go

```diff
@@ -74,7 +74,7 @@ type PFlash struct {
 func (arch Arch) PFlash(uefiEnabled bool, extraUEFISearchPaths []string) []PFlash {
 	switch arch {
 	case ArchArm64:
-		uefiSourcePaths := []string{"QEMU_EFI.fd", "/usr/share/OVMF/QEMU_EFI.fd"}
+		uefiSourcePaths := []string{"/usr/share/ovmf/OVMF.fd", "/usr/share/OVMF/QEMU_EFI.fd"}
```

Comment:
I'm not sure if reading firmware from the current working directory was intentional or not. If it was, we probably should decide if it's still desired, with the presence of the `--extra-uefi-search-paths` option.

Reply:
I think I missed, that, where was the use of the current working directory?

Reply:
brainfart, please ignore. I accidentially stripped the `/usr/share/qemu-efi-aarch64/` in the first commit.